pkgs/development/python-modules/pychromecast/default.nix

```diff
@@ -1,19 +1,21 @@
-{ lib, fetchurl, buildPythonPackage, requests, six, zeroconf, protobuf, casttube }:
+{ lib, fetchurl, buildPythonPackage, requests, six, zeroconf, protobuf, casttube, isPy3k }:
 
 buildPythonPackage rec {
   pname = "PyChromecast";
-  version = "2.3.0";
+  version = "2.4.0";
   name = pname + "-" + version;
 
   src = fetchurl {
     url    = "mirror://pypi/p/pychromecast/${name}.tar.gz";
-    sha256 = "f385168e34d2ef47f976c8e41bad2f58f5ca004634c0ccb1a12623d8beb2fa38";
+    sha256 = "0q012ghssk2xhm17v28sc2lv62vk7wd5p7zzdbgxk6kywfx8yvm2";
   };
 
+  disabled = !isPy3k;
+
   propagatedBuildInputs = [ requests six zeroconf protobuf casttube ];
```

Comment:
`six` is no longer required since https://github.com/balloob/pychromecast/commit/c15d23ee5d725eb5fb45dbbaa0593a9389c8ab38